R/gs_power_npe.R

```diff
@@ -200,6 +200,31 @@
 #'   upar = (x %>% filter(bound == "upper"))$z,
 #'   lpar = -(x %>% filter(bound == "upper"))$z
 #' )
+#'
+#' # Different values of `r` and `tol` leads to differ numerical accuracy
+#' # Larger `r` and smaller `tol` give better accuracy, but leads to slow computation
+#' n_analysis <- 5
+#' gs_power_npe(
+#'   theta = rep(0.1, n_analysis),
+#'   theta0 = NULL,
+#'   theta1 = NULL,
+#'   info = 1:n_analysis,
+#'   info0 = 1:n_analysis,
+#'   info1 = NULL,
+#'   info_scale = "h0_info",
+#'   upper = gs_spending_bound,
+#'   upar = list(sf = gsDesign::sfLDOF, total_spend = 0.025, param = NULL, timing = NULL),
+#'   lower = gs_b,
+#'   lpar = -rep(Inf, n_analysis),
+#'   test_upper = TRUE,
+#'   test_lower = FALSE,
+#'   binding = FALSE,
+#'   # Try different combination of (r, tol) with
+#'   # r in 6, 18, 24, 30, 35, 40, 50, 60, 70, 80, 90, 100
```

Comment:
What does the comments "r in [some numbers]" and "tol in [two numbers]" actually trying to say?